trampoline/src/test/java/com/iluwatar/trampoline/TrampolineAppTest.java

```diff
@@ -0,0 +1,17 @@
+package com.iluwatar.trampoline;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+public class TrampolineAppTest {
+
+    @Test
+    public void test()throws IOException{
```

Comment:
Please use descriptive test method name? What does it test?

Reply:
Done. Changed to testTrampolineWithFactorialFunction